src/index.tsx

```diff
@@ -8,6 +8,7 @@ import { BrowserRouter as Router } from "react-router-dom"
 
 const client = new ApolloClient({
   uri: "https://glowup-be.herokuapp.com/graphql",
+  // uri: "http://localhost:3001/graphql",
```

Comment:
It may be worth looking into a way to have the uri change depending on the current working environment (development / production)

Reply:
I agree! We can look more into it. Would love support if you have the bandwidth.